CardsForClimateUnityProj/Assets/Scripts/GameManager.cs

```diff
@@ -394,9 +399,9 @@ public void EndTurn()
         //reset counter
         activePlayerCardCount = 0;
         //reset values
```

Comment:
I wonder if we should encapsulate these value resets into a function, like `ResetValues()`?

Reply:
That's a good point! Probably wouldn't hurt if one of us wants to do it alongside finishing an issue somewhere down the line